Packages/Tests/SnapshotsTests/ConnectWalletSnapshotsTests.swift

```diff
@@ -28,10 +28,10 @@ final class ConnectWalletSnapshotsTests: XCTestCase {
                 reducer: { ConnectWalletReducer().dependency(\.analyticsClient, .consoleLogger) })
         )
             .environment(\.colorScheme, .dark)
-        XCTExpectFailure("Fails in github actions environment. Should pass locally")
+
         assertSnapshot(
             of: connectWalletView,
-            as: .image(perceptualPrecision: 0.98, layout: .device(config: .iPhoneSe)))
+            as: .image(perceptualPrecision: 0.9, layout: .device(config: .iPhoneSe)))
```

Comment:
0.9 is too low I guess. The reason I set XCTExpectFailure is that on some screenshots it was passing locally, but did not pass on server. I believe that either with newer snapshot testing lib or with different desings (pixel perfect to Figma) we might somehow get rid of these errors. I saw on one of your previous commits that one was fixed this way - so you might want to remove XCTExpectFailure on that screen, and keep others in place. XCTExpectFailure makes a test pass, even when it errors. And when the test passes, it yields error, so that you are warned that XCTExpectFailure can be removed.

Reply:
I found this related thread regarding this, but I haven't found a solution yet for our problem except for reducing the precision.

https://github.com/pointfreeco/swift-snapshot-testing/issues/810

Reply:
I prefer to integrate the tests with our CI rather than executing them on my local machine. This approach ensures I won't overlook running the tests before pushing changes.